sqlmesh/core/table_diff.py

```diff
@@ -223,15 +281,49 @@ def row_diff(self) -> RowDiff:
                     )
                     .limit(self.limit)
                 )
+                sample = self.adapter.fetchdf(sample_query)
+
+                joined_sample_cols = [f"s__{c}" for c in index_cols]
+                comparison_cols = [
+                    (f"s__{c}", f"t__{c}")
+                    for c in column_stats[column_stats["pct_match"] < 100].index
+                ]
+                for cols in comparison_cols:
+                    joined_sample_cols.extend([*cols])
```

Comment:
why [*cols] ?

Reply:
I was getting `TypeError: list.extend() takes exactly one argument (2 given)` with just `*cols`

Reply:
An idea is to define `comparison_cols` as a list of lists, so then you do `extend(cols)` directly here, I think.

Reply:
why not extend(cols) then?